cmd/api/main.go

```diff
@@ -15,6 +16,7 @@ import (
 
 func main() {
 	bootstrap.LoadConfig()
+	os.Setenv("TZ", "Asia/Bangkok")
```

Comment:
Could you please explain more about this?

Reply:
@Lahphim This is set to make sure that the default timezone is Bangkok. I set it on the test ENV as well to ensures that the test for the date helper does not fail on CI 😅